aries-site/src/data/structure.js

```diff
@@ -7,6 +7,23 @@ import {
   IconIcons,
   IconSquare,
 } from '../components';
+import {
+  AnchorPreview,
+  BoxPreview,
+  ButtonPreview,
+  CheckBoxPreview,
+  FooterPreview,
+  GridPreview,
+  HeaderPreview,
+  MaskedInputPreview,
+  MenuPreview,
+  RadioButtonGroupPreview,
+  RangeInputPreview,
+  SelectPreview,
+  TabsPreview,
+  TextAreaPreview,
+  TextInputPreview,
+} from '../examples/cardPreviews';
```

Comment:
Question for all @britt6612 @taysea related to project structure... is it more helpful to think of the "preview" belonging to the component example? Or to cardPreviews? If it belonged to the example, then I can easily see if that example has a preview or is missing it. Otherwise, I have to go and look for another directory which I may or may not know exists (assuming I am unfamiliar with the codebase).

In more concrete terms, would we rather have a structure like this:
/src
|--examples
|   |--components
|   |   |--button
|   |   |   |--ButtonExample.js
|   |   |   |--ButtonIconExample.js
|   |   |   |--ButtonPreview.js
|   |   |   |--index.js
|   |   |--checkbox
|   |   |   |--CheckboxExample.js
|   |   |   |--CheckboxPreview.js
|   |   |   |--index.js
|   |   ...
|   |--foundation
|   |--templates

Or like how it is now?
/src
|--examples
|   |--cardPreviews
|   |   |--ButtonPreview.js
|   |   |--CheckboxPreview.js
|   |   |--index.js
|   |   |  ...
|   |--components
|   |   |--button
|   |   |   |--ButtonExample.js
|   |   |   |--ButtonIconExample.js
|   |   |   |--index.js
|   |   |--checkbox
|   |   |   |--CheckboxExample.js
|   |   |   |--index.js
|   |   ...
|   |--foundation
|   |--templates

If we adopted the former, that work could be handled outside of this PR.

Reply:
Initially I did think about putting the Previews within the examples as you have above in the first example. Some of them I am using the examples since if they are simple enough so having them in the same section may be easier for a new user to realize here is a preview and here is the example then also by that know that if there is no preview then the example was most likely used

Reply:
I kind of like the first option as well, easier for scan-ability. I agree, I'd be fine updating structure in a separate PR.

Reply:
Ill create an issue for re structuring these files!